### ISSUE TITLE
Build fails resolving transitive dependency

### ISSUE THREAD

**Issue by es0329** [arunkumar9t2/scabbard#92]
Would [a newer version of kittinunf/Result](https://mvnrepository.com/artifact/com.github.kittinunf.result/result) in scabbard solve this problem?

```txt
> Task :app:kaptGenerateStubsDevDebugKotlin FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:kaptGenerateStubsDevDebugKotlin'.
> Could not resolve all files for configuration ':app:kapt'.
   > Could not find com.github.kittinunf.result:result:3.0.0.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/github/kittinunf/result/result/3.0.0/result-3.0.0.pom
       - https://repo.maven.apache.org/maven2/com/github/kittinunf/result/result/3.0.0/result-3.0.0.pom
       - https://kotlin.bintray.com/kotlinx/com/github/kittinunf/result/result/3.0.0/result-3.0.0.pom
       - https://oss.sonatype.org/content/repositories/snapshots/com/github/kittinunf/result/result/3.0.0/result-3.0.0.pom
     Required by:
         project :app > dev.arunkumar:scabbard-processor:0.5.0
```

**Comment by arunkumar9t2** [comment]
Ah, I will update to latest version and do a hotfix release.

**Comment by paynd** [comment]
Still happens in 0.5.0, should I use 0.5.1?

**Comment by arunkumar9t2** [comment]
Hey @paynd, this is fixed in `main` branch but `0.5.1` is not released it. In the mean time, please use a custom resolution strategy to force gradle to use 3.0.1 version of the transitive dependency like below

```groovy
    configurations.all {
        resolutionStrategy.eachDependency {
            if (requested.group == "com.github.kittinunf.result" && requested.name == "result" && requested.version == "3.0.0") {
                useVersion("3.0.1")
                because("Transitive dependency of Scabbard, currently not available on mavenCentral()")
            }
        }
    }
```

I added an example PR here: https://github.com/arunkumar9t2/scabbard-sample/pull/1/files

**Comment by paynd** [comment]
Yep, it works!
